tests/fixture_valid.js

```diff
@@ -192,47 +129,41 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
       }
 
       if ('capabilities' in channel) {
-        for (let i=0; i<channel.capabilities.length; i++) {
-          const cap = channel.capabilities[i];
+        for (let i = 1; i <= channel.capabilities.length; i++) {
+          const cap = channel.capabilities[i-1];
 
+          // range valid / no overlapping
           if (cap.range[1] >= dmxMaxBound) {
             result.errors.push({
               description: `range values must be strictly less than ${dmxMaxBound} in capability #${i} in channel '${ch}'.`,
               error: null
             });
-            break;
```

Comment:
If the first range is invalid, there's no use to make the other range tests. That's why we simply use `else if`. But it doesn't make sense to skip color / image / switching channel tests only because the range is wrong.